UnitsNet.Tests/CustomCode/TemperatureDeltaTests.cs

```diff
@@ -52,5 +52,14 @@ public class TemperatureDeltaTests : TemperatureDeltaTestsBase
         protected override double DegreesReaumurDeltaInOneKelvinDelta => 0.8;
         protected override double DegreesRoemerDeltaInOneKelvinDelta => 21/40d;
         protected override double KelvinsDeltaInOneKelvinDelta => 1;
+
+        protected override double DegreesCelsiusInOneKelvinDelta => 1;
+        protected override double DegreesDelisleInOneKelvinDelta => -1.5d;
+        protected override double DegreesFahrenheitInOneKelvinDelta => 1.8;
+        protected override double DegreesNewtonInOneKelvinDelta => 0.33d;
+        protected override double DegreesRankineInOneKelvinDelta => 1.8;
+        protected override double DegreesReaumurInOneKelvinDelta => 0.8;
+        protected override double DegreesRoemerInOneKelvinDelta => 21 / 40d;
+        protected override double KelvinsInOneKelvinDelta => 1;
```

Comment:
When changing base unit to Kelvin as commented below and regenerating the code, then this test code must be updated too.